gateway-incoming.yaml

```diff
@@ -146,6 +154,36 @@ paths:
               #! end !#
               "bodyPassthrough": true
             }
+  /trigger-sync:
+    post:
+      description: "Trigger an asynchronous job to synchronize the IPAM data from Device42 with the IPAM Facade database"
+      responses:
+        202:
+          description: "Accepted"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/JobMetadata"
+        500:
+          description: "Triggger failed. Sync operation will not be performed."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-transportd:
+        backend: app
+        enabled:
+          - "metrics"
+          - "accesslog"
+          - "requestvalidation"
+          - "responsevalidation"
+          - "lambda"
+        lambda:
+          arn: "enqueue"
+          async: false
+          request: '{}'
```

Comment:
Question: why hardcode an empty body?

Reply:
`#! json .Request.Body !#` is how you'd do a body passthrough in the request template if that's wht you need.